static/app/actionCreators/integrations.tsx

```diff
@@ -13,16 +13,16 @@ const api = new Client();
 /**
  * Removes an integration from a project.
  *
- * @param {String} orgId Organization Slug
- * @param {String} projectId Project Slug
- * @param {Object} integration The organization integration to remove
+ * @param orgSlug Organization Slug
+ * @param projectId Project Slug
+ * @param integration The organization integration to remove
  */
 export function removeIntegrationFromProject(
-  orgId: string,
+  orgSlug: string,
```

Comment:
sorry, i just don't like orgId

Reply:
I'm a fan of `orgSlug` as well, more of this :clap: